blog/templates/blog/blog_index_page.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% load pageurl richtext include_block from wagtailcore_tags %}
+
+{% block main %}
+
+	<h1 class="page-title">
+		Blog
+	</h1>
+
+	{% if page.body %}
+		<div class="site-summary site-summary--blog">
+			{% for block in page.body %}
+				{% include_block block %}
```

Comment:
I noticed this too. I am not sure about the design here since in the design it's always just a paragraph. I guess I can add `max-width: 100%` anyways to give some sane output.